csharp/ql/src/semmle/code/csharp/frameworks/microsoft/AspNetCore.qll

```diff
@@ -191,11 +199,11 @@ class MicrosoftAspNetCoreMvcController extends Class {
   }
 }
 
-/** The `Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper` class. */
-class MicrosoftAspNetCoreMvcHtmlHelperClass extends Class {
-  MicrosoftAspNetCoreMvcHtmlHelperClass() {
-    getNamespace() instanceof MicrosoftAspNetCoreMvcViewFeatures and
-    hasName("HtmlHelper")
+/** The `Microsoft.AspNetCore.Mvc.Rendering.IHtmlHelper` interface. */
+class MicrosoftAspNetCoreMvcRenderingHtmlHelperInterface extends Interface {
```

Comment:
Should be called `MicrosoftAspNetCoreMvcRenderingIHtmlHelperInterface` (extra `I`)